shared-services/src/files/cloudinary-file-uploader.ts

```diff
@@ -0,0 +1,46 @@
+import { CloudinaryUpload } from "@bloom-housing/ui-components"
+import { AssetsService } from "@bloom-housing/backend-core/types"
+
+export class CloudinaryFileUploader {
```

Comment:
This change is so that the method to upload a file that is in this class can be injected via this class so we can mock it for testing. Also, refactoring this into shared-services removes the dependency on the partners site since that's where the code lived before.